litex/soc/software/libbase/strtod.c

```diff
@@ -166,7 +166,7 @@ double strtod(const char *str, char **endptr)
       switch(*++p)
         {
         case '-':
-          negative = 1;   /* Fall through to increment pos */
+          negative = 1;   /* FALLTHRU */ /* to increment pos */
```

Comment:
Are you sure you wanted this to be part of the PR?

Reply:
See previous response.